Tests/DangerTests/GitDiffTests.swift

```diff
@@ -52,28 +71,34 @@ final class GitDiffTests: XCTestCase {
                                    FileDiff.Line(text: "    enum Errors: Error {", changeType: .unchanged),
                                ]),
-                               .init(oldLineStart: 22, oldLineSpan: 6, newLineStart: 25, newLineSpan: 10, lines: [
+                                        .init(oldLineStart: 22, oldLineSpan: 6, newLineStart: 25, newLineSpan: 10, lines: [
                                    FileDiff.Line(text: "        let coverage = try JSONDecoder().decode(SPMCoverage.self, from: data)", changeType: .unchanged),
-                                   FileDiff.Line(text: "        let filteredCoverage = coverage.filteringFiles(notOn: files)", changeType: .unchanged),
-                                   FileDiff.Line(text: "", changeType: .unchanged),
-                                   FileDiff.Line(text: "        return Report(messages: [], sections: [ReportSection(fromSPM: filteredCoverage, fileManager: fileManager)])", changeType: .removed),
-                                   FileDiff.Line(text: "        if filteredCoverage.data.contains(where: { !$0.files.isEmpty }) {", changeType: .added),
-                                   FileDiff.Line(text: "            return Report(messages: [], sections: [ReportSection(fromSPM: filteredCoverage, fileManager: fileManager)])", changeType: .added),
-                                   FileDiff.Line(text: "        } else {", changeType: .added),
-                                   FileDiff.Line(text: "            return Report(messages: [], sections: [])", changeType: .added),
-                                   FileDiff.Line(text: "        }", changeType: .added),
-                                   FileDiff.Line(text: "    }", changeType: .unchanged),
-                                   FileDiff.Line(text: "}", changeType: .unchanged),
-                               ]),
-                           ]),
-                           FileDiff(parsedHeader: .init(filePath: "Sources/DangerSwiftCoverage/ShellOutExecutor1.swift", change: .renamed(oldPath: "Sources/DangerSwiftCoverage/ShellOutExecutor.swift")), hunks: [
+                                            FileDiff.Line(text: "        let filteredCoverage = coverage.filteringFiles(notOn: files)", changeType: .unchanged),
+                                            FileDiff.Line(text: "", changeType: .unchanged),
+                                            FileDiff.Line(text: "        return Report(messages: [], sections: [ReportSection(fromSPM: filteredCoverage, fileManager: fileManager)])", changeType: .removed),
+                                            FileDiff.Line(text: "        if filteredCoverage.data.contains(where: { !$0.files.isEmpty }) {", changeType: .added),
+                                            FileDiff.Line(text: "            return Report(messages: [], sections: [ReportSection(fromSPM: filteredCoverage, fileManager: fileManager)])", changeType: .added),
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitDiffTests.swift;
  Line: 89;
-->

- :warning: Line should be 140 characters or less: currently 207 characters (`line_length`)